modules/mixed-arch-modify-machine-set.adoc

```diff
@@ -0,0 +1,69 @@
+//Module included in the following assembly
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-modify-machine-set_{context}"]
+
+= Adding the `arm64` bootimage to your machine set 
+
+To deploy the heterogeneous cluster, you must add the generated `arm64` bootimage to your Azure machine set. To create your own custom machine set on Azure, see "Creating a machine set on Azure".
+
+.Prerequisites 
+
+* Installed the OpenShift CLI (`oc`).
+* Generated a custom Azure machine set. 
+
+.Procedure
+. Check that your machine set was created:
++
+[source,terminal]
+----
+oc get machineset
+----
+. Add your generated `arm64` bootimage to the `resourceID` parameter, and then edit `vmSize` to the instance type used in your installation: 
++
+[source,terminal]
+----
+oc edit machineset <machineset> 
+----
+.Sample YAML machine set with `arm64` bootimage 
++
+[source,yaml]
+----
+    providerSpec:
+        value:
+          apiVersion: azureproviderconfig.openshift.io/v1beta1
+          credentialsSecret:
+            name: azure-cloud-credentials
+            namespace: openshift-machine-api
+          image:
+            resourceID: /resourceGroups/<infrastructure_id>-rg/providers/Microsoft.Compute/images/<infrastructure_id> 
```

Comment:
Hi @kelbrown20 the resourceID should be the same one in [mixed-arch-creating-arm-boot, line 123](https://github.com/openshift/openshift-docs/pull/46224/files#diff-545a67ca9198058b1e5d769b9e06454b07b35642d001de812e4494c76ce37946R123).